lib/attrsets.nix

```diff
@@ -333,6 +333,40 @@ rec {
       ) (attrNames set)
     );
 
+   /*
+    Like builtins.foldl' but for Attrsets.
+    Iterates over every nameValuePair in the given AttrSet.
+    The result can be appended to `acc` which gets finally returned.
+
+    Attention:
+      There is a completely different function
+      'lib.foldAttrs'
+      which has a very similiar name, but is very different.
+
+    Example:
+      foldlAttrs
+        ({ sum, names }: name: value: {
```

Comment:
pattern matching on acc might be confusing. For first time readers ?
So should i write example:
```
(acc: name: value: {
sum = acc.sum + value
names = acc.names ++ [name];
}
```

Reply:
lgtm